code/modules/martial_arts/combos/judo/eyepoke.dm

```diff
@@ -0,0 +1,17 @@
+/datum/martial_combo/judo/eyepoke
+	name = "Eye Poke"
+	steps = list(MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_HARM)
+	explaination_text = "Jab your opponent in the eye, damaging and blinding them. Minimally effective on people with eye protection."
```

Comment:
Is this second part actually true, or do you just mean with regards to the flash?

Reply:
This is just in regards to the flash. This is going to be removed in favor of the eyeblind that WILL work through glasses, but not give eye damage, Inline with Matt's review.